services/core/shared/core/compat/sdk_v5/accounts.js

```diff
@@ -321,6 +321,25 @@ const getAccountsByPublicKey = async (accountInfoArray) => {
 	return accounts;
 };
 
+const getAccountsByPublicKey2 = async (accountInfoArray) => {
```

Comment:
Do we need this method?
The method `getAccountsByPublicKey` has the exact same logic except that `accountInfoArray` is an array of objects containing the `publicKey` property.

Reply:
Technically not, however this will require to do more refactoring which is beyond the scope of that issue.